evm_loader/program/src/executor.rs

```diff
@@ -280,20 +281,21 @@ impl<'a, B: AccountStorage> Handler for Executor<'a, B> {
         Capture::Trap(CreateInterrupt{context, transfer, address, init_code})
     }
 
+    #[allow(unused_variables)]
```

Comment:
Keep it more local. 
`#[allow(unused_variables)] target_gas: Option<u64>, `

Reply:
fixed